src/table/utils/handle-key-press.js

```diff
@@ -134,10 +134,17 @@ export const bodyHandleKeyPress = ({
   selectionsAPI = null,
 }) => {
   const isInSelectionMode = selectionsAPI?.isModal();
+  const isMultiValueSelections = () =>
+    (cell.prevQElemNumber || cell.nextQElemNumber) && evt.shiftKey && cell.isSelectable && isAnalysisMode;
```

Comment:
Now this will make an engine call going up to the header as well... You could check them individually instead:
```suggestion
    evt.shiftKey && cell.isSelectable && isAnalysisMode && (cell.prevQElemNumber && evt.key === 'ArrowUp' || cell.nextQElemNumber && evt.key === 'ArrowDown') ;
```

Reply:
Good point. Thanks. Fixed